doc/content/lorawan/concentrators/_index.md

```diff
@@ -0,0 +1,95 @@
+---
+title: "LoRaWAN® Concentrators"
+section: "Additional Information"
+description: ""
+weight:
+---
+
+[Semtech's LoRa concentrators](https://www.semtech.com/products/wireless-rf/lora-core) power all LoRa communication.
+
+Semtech has produced four LoRa concentrators and all LoRaWAN gateways use one of these devices to receive and transmit LoRa messages.
+
+Read about all of Semtech's LoRa products on their [LoRa product page](https://www.semtech.com/products/wireless-rf/lora-core).
+
+Additionally, RAK Wireless provides a great [breakdown of available LoRa hardware](https://news.rakwireless.com/things-to-consider-when-picking-a-lorawan-gateway/).
+
+This page provides a quick description of each of the LoRa concentrators.
+
+## SX1301
+
+{{< figure src="SX1301.jpg" class="float plain" >}}
+
+The SX1301 is Semtech's first outdoor LoRaWAN concentrator, designed to provide worldwide LoRaWAN gateway capability in ISM bands. It is the first SoC to integrate Semtech's LoRa Concentrator IP.
```

Comment:
The SX1301 is not really a SoC as it does not contain a user programmable CPU/MCU. Does Semtech call it a SoC?

Reply:
How about "... is the first digital baseband chip to integrate Semtech's..."?